inventory-items.go

```diff
@@ -0,0 +1,40 @@
+package shopify
+
+// InventoryItemRepository manages inventory items
+type InventoryItemRepository interface {
+	// Get retrieves an inventory item by its id
+	Get(id int64) (InventoryItem, error)
+}
+
+type CountryHarmonizedSystemCode struct {
+	HarmonizedSystemCode string
+	CountryCode          string
+}
+
+// InventoryLevel represents the amount of an item that there is available in a shop
+type InventoryItem struct {
+	// The ID of the inventory item
+	ID int64
+	// The unique SKU (stock keeping unit) of the inventory item
+	SKU string
+	// The date and time when the inventory level was created
+	CreatedAt string
+	// The date and time when the inventory level was last modified
+	UpdatedAt string
+	// Whether a customer needs to provide a shipping address when placing an order containing the inventory item
+	RequiresShipping bool
+	// The unit cost of the inventory item
+	Cost string
+	// The country code of where the item came from (ISO)
+	CountryCodeOfOrigin string
+	// The province code of where the item came from (ISO)
+	ProvinceCodeOfOrigin string
+	// The general Harmonized System (HS) code for the inventory item. Used if a country-specific HS code (`countryHarmonizedSystemCode`) is not available.
+	HarmonizedSystemCode int
```

Comment:
```suggestion
	HarmonizedSystemCode int64
```
Should this be int64?